src/cursor.c

```diff
@@ -290,33 +290,32 @@ int Cursor_Free(Cursor *cur) {
   return Cursors_Purge(cur->parent, cur->id);
 }
 
-void Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
+int Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
   CursorList_Lock(cl);
   CursorSpecInfo *info = findInfo(cl, name, NULL);
-  size_t n = 0;
+  if (!info) {
```

Comment:
@ashtul how can this even happen? each index must have a cursor list associated with it no?